README.md

```diff
@@ -74,42 +74,27 @@ We also add following methods to make type-checking more easily:
 - [`pluck_to_tstruct`](#pluck_to_tstruct-instead-of-pluck)
 
 ### Controllers
-```sh
-❯ rake rails_rbi:custom
```

Comment:
We still need this I think. I prefer to add `typed_params.rbi` to `bundled_rbi`. You'll also need to add the file to rails_rbi rake task to be copied with `rake rails_rbi:custom`